.github/workflows/test-windows.yml

```diff
@@ -0,0 +1,56 @@
+name: test-windows
+on:
+  pull_request:
+    paths-ignore:
+      - "**.md"
+      - "**.jpg"
+      - "**.png"
+      - "**.gif"
+      - "**.svg"
+      - "adr/**"
+      - "docs/**"
+
+# Abort prior jobs in the same workflow / PR
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  validate:
+    runs-on: windows-latest
```

Comment:
Can you change this and spell out the version of Windows explicitly. The window-latest label can change.

Reply:
I think this one might be okay as it changes pretty infrequently in github image land and is how we do all other CI rn (with ubuntu-latest).